README.md

```diff
@@ -11,6 +11,8 @@ continuously updating weather data on top of a responsive map, providing the
 capability to monitor weather events using reflectivity, velocity, and other
 products.
 
+Please be sure to check out the documentation before getting started: [Supercell-Wx Documentation](https://supercell-wx.rtfd.io/)
```

Comment:
"Supercell-Wx" should be stylized as "Supercell Wx"

Reply:
Fixed the styling here and in another location on the doc.